index.html

```diff
@@ -66,99 +76,103 @@ <h2>My Recent Works</h2>
-          <ul class="cardBtns">
-            <li><button class="cardBtn">HTML</button></li>
-            <li><button class="cardBtn">BootStrap</button></li>
-            <li><button class="cardBtn">Ruby</button></li>
-          </ul>
+        <!-- Card 3-->
+        <div class="cardContainer">
+          <div class="card">
+            <h2>Profesional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="cardBtns">
+              <li><button class="cardBtn">HTML</button></li>
+              <li><button class="cardBtn">BootStrap</button></li>
+              <li><button class="cardBtn">Ruby</button></li>
+            </ul>
+          </div>
+          <button class="cardSeeProj">See Project</button>
         </div>
-        <button class="cardSeeProj">See Project</button>
-      </div>
-      <!-- Card 1-->
-      <div class="cardContainer">
-        <div class="card">
-          <h2>Profesional Art <br />Printing Data</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <ul class="cardBtns">
-            <li><button class="cardBtn">HTML</button></li>
-            <li><button class="cardBtn">BootStrap</button></li>
-            <li><button class="cardBtn">Ruby</button></li>
-          </ul>
+        <!-- Card 4-->
+        <div class="cardContainer">
+          <div class="card">
+            <h2>Profesional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="cardBtns">
+              <li><button class="cardBtn">HTML</button></li>
+              <li><button class="cardBtn">BootStrap</button></li>
+              <li><button class="cardBtn">Ruby</button></li>
+            </ul>
+          </div>
+          <button class="cardSeeProj">See Project</button>
+        </div>
+        <!-- Card 5-->
+        <div class="cardContainer">
+          <div class="card">
+            <h2>Profesional Art <br />Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="cardBtns">
+              <li><button class="cardBtn">HTML</button></li>
+              <li><button class="cardBtn">BootStrap</button></li>
+              <li><button class="cardBtn">Ruby</button></li>
+            </ul>
+          </div>
+          <button class="cardSeeProj">See Project</button>
         </div>
-        <button class="cardSeeProj">See Project</button>
       </div>
     </section>
     <section class="about">
```

Comment:
I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220550557-4324b6a6-f658-4a43-aa19-01a6ecbc768a.png)


## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/220550575-299126ca-98b9-41b2-8927-c287cf54548e.png)

## Remember:-
You did great work but the project demands to make it as close as possible to the design almost 100% exact. So your design font-sizes, margins, padding, width, and height are actually similar to the design.